helper/load_test.go

```diff
@@ -30,15 +30,15 @@ func setup() (synq.ApiV2, Cache) {
 	os.MkdirAll(cacheDir, 0755)
 	cache := Cache{Dir: cacheDir}
 	api := synq.NewV2(testAuth)
-	server := test_server.SetupServer("v2", sampleDir)
+	server := test_server.SetupServer(SYNQ_VERSION, sampleDir)
 	url := server.GetUrl()
 	api.SetUrl(url)
 	api.UploadUrl = url
 	return api, cache
 }
 
 func (c Cache) GetCacheFile(name string) string {
-	return c.Dir + "/" + name + ".json"
+	return "./" + c.Dir + "/" + name + ".json"
```

Comment:
c.Dir could be a "full path", so putting a "./" in front would be wrong.  Any particular reason why this is set?

Reply:
When I do testing, the path without "./" would return false in os.Stat(). I don't know exactly why.

Reply:
The original code works properly now. Might be some other reasons caused the error in previous test. I just changed it back.